src/RecoveryServices/RecoveryServices.SiteRecovery/ReplicationProtectedItem/SetAzureRmRecoveryServicesAsrReplicationProtectedItem.cs

```diff
@@ -342,6 +374,10 @@ public override void ExecuteSiteRecoveryCmdlet()
                 var availabilityZone = this.RecoveryAvailabilityZone;
                 var primaryNic = this.PrimaryNic;
                 var diskIdToDiskEncryptionMap = this.DiskIdToDiskEncryptionSetMap;
+                var sqlServerLicenseType = this.SqlServerLicenseType;
+                var recoveryVmTag = this.RecoveryVmTag;
+                var recoveryNicTag = this.RecoveryNicTag;
+                var diskTag = this.DiskTag;
```

Comment:
>DiskTag [](start = 35, length = 7)

What if for H2A useManagedDisk is false? SRS will throw exception so are we fine with that? or we want to throw from powershell

Reply:
Handled in powershell.

---
In reply to: [601551907](https://github.com/AsrOneSdk/azure-powershell/pull/21#discussion_r601551907) [](ancestors = 601551907)